esm-samples/jsapi-vue/package.json

```diff
@@ -7,11 +7,11 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@arcgis/core": "~4.24.0",
-    "vue": "^3.2.37"
+    "@arcgis/core": "^4.25.0",
```

Comment:
Did you mean to change these from ~ to ^?

Reply:
Nope, that was an accident. Fixed.

Reply:
There's still three places using "^4.25.0"

Reply:
Argh, relearning how to use project search, lol, I found and fixed.